tests/integration/interactive/cmd_init_test.go

```diff
@@ -59,4 +61,126 @@ var _ = Describe("odo init interactive command tests", func() {
 		Expect(output).To(ContainSubstring("Your new component \"my-go-app\" is ready in the current directory."))
 		Expect(helper.ListFilesInDir(commonVar.Context)).To(ContainElements("devfile.yaml"))
 	})
+
+	Describe("Displaying welcoming messages", func() {
+
+		type tester func(*expect.Console, *bytes.Buffer)
+
+		testFunc := func(language string, welcomingMsgs []string, additionalTester tester) tester {
```

Comment:
Nice. Can you document what's the behaviour of this function so it could be reused or we can use it at a template for other tests?
Something similar to:
```
// this function returns a tester (usable for `RunInteractive`), that first expects welcomingMsgs, then run additionalTesters and finally expects the asking of a component name
```